### ISSUE TITLE
Default sort order toggle in Content mode

### ISSUE THREAD

**Issue by salvor-hardin** [Sequel-Ace/Sequel-Ace#1374]
When you open the Content view for a table/view and click on any column name, the default is to sort ascending the results based on that column.

Would it be possible to have an option to toggle that to sort **descending** first by default?

I find myself always needing to sort descending so I have to click twice there. It would be useful if the default sorting (ascending/descending) could be an option in Settings so each user can adjust based on their needs.

Thanks

**Comment by samuelbradshaw** [comment]
See https://github.com/Sequel-Ace/Sequel-Ace/issues/1267:

> If you hold shift while clicking the column header, it'll go to DESC immediately.

macOS Finder in list view provides an interesting alternative – it sorts ascending or descending by default depending on the data type. If it's text, the default is ascending. If it's a date, the default is descending.

**Comment by GS-Giuliano** [comment]
Sort ascending as default doesn't make sense in production environment. The user always want to see last records.
Please think about that...

**Comment by Jason-Morcos** [comment]
> Sort ascending as default for integer autoincrement fields doesn't make sense in production environment. The user always want to see last records.
> 
> Please think about that...

I don't agree on this one. I think sorting ascending can make a lot of sense depending on what you're looking at.
For example, if you had a table where you stored requested jobs, you might want to always see them in order of when they are requested (so oldest first). Not to mention alphabetical sorting if you've filtered down and are looking at a small list of things.

I think switching to always default descending isn't an option.

The option to have certain columns default ascending/descending depending on the data type is an option, or a default to descending setting option works, and maybe even better the column could "remember" the last way it was sorted and go back to that sort on first click.

**Comment by Jason-Morcos** [comment]
Also, see this - https://github.com/Sequel-Ace/Sequel-Ace/issues/1267

Shift click goes straight to descending

**Comment by salvor-hardin** [comment]
The ask is to offer an option in settings to toggle the default sort… that’s all. Shift holding works, but is not optimal for users that want always to see descending.
Thanks all for the inputs !

**Comment by GS-Giuliano** [comment]
I agree @salvor-hardin. That option works for all scenarios.
@Jason-Morcos thanks for the tip.

**Comment by jdwuzhere** [comment]
Maybe we should save a sort order per table and reuse it when reopening this table content viewer?

**Comment by tyzion** [comment]
I'd like to have this option too 👍🏼
